config/default.yml

```diff
@@ -705,6 +705,12 @@ Rails/Validation:
   Include:
     - app/models/**/*.rb
 
+Rails/WhereEquals:
+  Description: 'Pass conditions to `where` as a hash instead of manually constructing SQL.'
+  StyleGuide: 'https://rails.rubystyle.guide/#hash-conditions'
```

Comment:
This doesn't exist yet: https://github.com/rubocop-hq/rails-style-guide/pull/270 should be reviewed first.

Reply:
https://github.com/rubocop-hq/rails-style-guide/pull/270 was merged! ✅